New pages: HTML<formelements>Element.value properties (#35641)
mdn/content: files/en-us/web/api/htmltextareaelement/value/index.md

---
title: "HTMLTextAreaElement: value property"
short-title: value
slug: Web/API/HTMLTextAreaElement/value
page-type: web-api-instance-property
browser-compat: api.HTMLTextAreaElement.value
---

{{ APIRef("HTML DOM") }}

> The **`value`** property of the {{DOMxRef("HTMLTextAreaElement")}} interface represents the value of the {{htmlelement("textarea")}} element as a string, which is an empty string if the widget contains no content. It reflects the raw value contained in the control.

Reviewer: "Reflect attribute `x`" is a proper term in HTML, which means it acts as a proxy for getting/setting the attribute value. Therefore you don't use "reflect" for anything other than attributes. (You can know whether a property reflects an attribute by checking the spec.)

Author: changed to "returns or sets"

> The **`value`** property of the {{DOMxRef("HTMLTextAreaElement")}} interface represents the value of the {{htmlelement("textarea")}} element as a string, which is an empty string if the widget contains no content. It reflects the raw value contained in the control.

Author: ```suggestion
The **`value`** property of the {{DOMxRef("HTMLTextAreaElement")}} interface represents the value of the {{htmlelement("textarea")}} element as a string, which is an empty string if the widget contains no content. It returns or sets the raw value contained in the control.
```

> 
> This property can also be set directly, for example to set or update the value based on some condition.

Author: ```suggestion
```

## Value

> A string containing the contents of the {{htmlelement("textarea")}} element, .

Reviewer: ```suggestion
A string containing the contents of the {{htmlelement("textarea")}} element.
```
